Makefile

```diff
@@ -313,9 +315,9 @@ endif
 # Build a bundle image using the 'operator-sdk' tool
 .PHONY: bundle
 bundle: operator-sdk manifests kustomize ## Generate bundle manifests and metadata, then validate generated files.
-	$(OPSDK) generate kustomize manifests -q
+	#$(OPSDK) generate kustomize manifests -q
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
-	$(KUSTOMIZE) build config/manifests | $(OPSDK) generate bundle $(BUNDLE_GEN_FLAGS)
+	$(KUSTOMIZE) build config/manifests/$(PROFILE) | $(OPSDK) generate bundle --kustomize-dir config/manifests/$(PROFILE) $(BUNDLE_GEN_FLAGS)
```

Comment:
The `bundle` directory now corresponds to the bundle of the default profile, which might be confusing to me (e.g., we now have 2 `{backstage,rhdh}-controller-manager-metrics-service_v1_service.yaml` files in the `bundle/manifests` dir - see https://github.com/gazarenkov/rhdh-operator/tree/external-default-config/bundle/manifests).
Just like we have different `config/profile` folders, would it make sense to have different `bundle` directories per profile?

Reply:
Even if I still do not think it is important for end user, it somehow helpful for us to see and understand what we generate on this stage :), so I fixed that.
@rm3l I changed the file path but diff checker failed trying to read it on the old path by some reason, could you please take a look.

Reply:
> I changed the file path but diff checker failed trying to read it on the old path by some reason, could you please take a look.

Yeah, I think that's because the Workflow is triggered on a `pull_request_target` event (to prevent PRs from forks stealing the GH secrets needed to commit/push to external fork PRs). So the content of the Workflow is loaded from the base branch, even if you changed it in your PR.
Anyway, I think it is fine right now. I could confirm manually that your change to the Workflow works correctly.

/override "PR Bundle Diff "

Reply:
/override "PR Bundle Diff"